faculty/clients/experiment.py

```diff
@@ -645,24 +656,33 @@ def restore_runs(self, project_id, run_ids=None):
         project_id : uuid.UUID
         run_ids : List[uuid.UUID], optional
             A list of run IDs to restore. If not specified, all runs in the
-            project will be restored.
+            project will be restored. If an empty list is passed, no runs
+            will be restored.
 
         Returns
         -------
         RestoreExperimentRunsResponse
             Containing lists of successfully restored and conflicting (already
             active) run IDs.
         """
+        endpoint = "/project/{}/run/restore/query".format(project_id)
 
-        query_params = []
+        payload = {}
```

Comment:
What would you think about having something like:

```python
if run_ids is None:
    # Delete all runs in project
    payload = {}  # No filter
elif len(run_ids) == 0:
    return RestoreExperimentRunsResponse(
        restored_run_ids=[], conflicted_run_ids=[]
    )
else:
    payload = {
        "filter": {
            "operator": "or",
            "conditions": [
                {"by": "runId", "operator": "eq", "value": str(run_id)}
                for run_id in run_ids
            ],
        }
    }
```

This would remove a level of nesting plus make it more explicit how `run_ids=None` results in deleting all runs of the project.

Reply:
👍